lib/matplotlib/backends/backend_gdk.py

```diff
@@ -441,6 +441,12 @@ def _renderer_init(self):
         self._renderer = RendererGDK (gtk.DrawingArea(), self.figure.dpi)
 
     def _render_figure(self, pixmap, width, height):
+        if isinstance(self._renderer, RendererGDK):
+            warn_deprecated('2.0', message="The GDK backend is "
+                            "deprecated. It is untested, known to be "
+                            "broken and will be removed in Matplotlib 2.2. "
+                            "Use the Agg backend instead.",
```

Comment:
I don't think you need this line; the replacement advice is auto-generated when you supply the alternative kwarg.